### PR TITLE
Giftcard Hub API Spectral errors fix

### PR DESCRIPTION
**Pull request by ricardoaerobr**
#### Types of changes
- [ ] New content (endpoints, descriptions or fields from scratch)
- [ ] Improvement (make an endpoint's title or description even better)
- [X] Spelling and grammar accuracy (self-explanatory)

### Changelog
Do not forget to update your changes to our Developer Portal's changelog. Did you create a release note?
- [ ] Yes, I already created a release note about this change.
- [X] No, but I am going to.

PR created to fix spectral errors (Giftcard Hub API) - EDU 12751

### CODE REVIEW

File: VTEX - GiftCard Hub API.json [hunk 1]
```diff
@@ -407,28 +407,20 @@
...
                     "200": {
-                        "description": "OK",
-                        "content": {
-                            "application/json": {
-                                "schema": {
-                                    "type": "object"
-                                }
-                            }
-                        }
+                        "description": "OK. This endpoint does not return any data in the response body."
```
Comment (PedroAntunesCosta) [inline VTEX - GiftCard Hub API.json:410]:
```suggestion
                        "description": "OK"
```

Comment (PedroAntunesCosta) [reply VTEX - GiftCard Hub API.json:410]: Response descriptions should only be the status code name according to our [template](https://github.com/vtex/openapi-schemas/blob/master/templates/VTEX%20-%20Template%20openAPI.jsonc).

File: VTEX - GiftCard Hub API.json [hunk 2]
```diff
@@ -407,28 +407,20 @@
...
                     },
                     "404": {
-                        "description": "The gift card provider described does not exist.\n"
+                        "description": "The gift card provider described does not exist."
```
Comment (PedroAntunesCosta) [inline VTEX - GiftCard Hub API.json:413]:
```suggestion
                        "description": "Not Found"
```

File: VTEX - GiftCard Hub API.json (hunk 1 above)
Comment (ricardoaerobr) [reply VTEX - GiftCard Hub API.json:410]: We had previously discussed this message for responses where it is necessary to make it clear to the customer that 200-OK means success and there is no content to be displayed.

We should have updated the API template at that time.

Thread: https://vtex.slack.com/archives/C05G6ANSU67/p1701085906819989?thread_ts=1700245355.794799&cid=C05G6ANSU67

**Review by PedroAntunesCosta — APPROVED** [review]
LGTM

**Pull request merged**